src/Checkout/CheckoutScreen.js

```diff
@@ -0,0 +1,301 @@
+import React, { useEffect, useState } from 'react';
+import {
+  View, StyleSheet, Alert, ScrollView,
+} from 'react-native';
+import {
+  Text, Button,
+} from 'react-native-paper';
+import PropTypes from 'prop-types';
+import Config from 'react-native-config';
+import Icon from 'react-native-vector-icons/Ionicons';
+
+import CartProduct from '../Cart/CartProduct';
```

Comment:
The cart product cards in the checkout screen are a little different than Kaylee's cart product, there shouldn't be any line separators and you shouldn't be able to modify the weight/quantity. It might be easier to create a new Cart Product based on Kaylee's that have those modifications, or you could talk to Kaylee to toggle those modifications.